### PR TITLE
Acl path

### PR DESCRIPTION
**Pull request by LubosRemplik**
Useful when you want locate acl.ini in your plugin for example. What do you think?

### CODE REVIEW

**Comment by dereuromark** [comment]
I am fine with that.

File: docs/README.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ prefixes|array|A list of authorizeByPrefix handled prefixes.
 allowUser|bool|True will give authenticated users access to all resources except those using the `adminPrefix`
 adminPrefix|string|Name of the prefix used for admin pages. Defaults to admin.
 autoClearCache|bool|True will generate a new ACL cache file every time.
+aclPath|string|Full path to the acl.ini. Default ``ROOT . DS . 'config' . DS``.
```
Comment (dereuromark) [inline docs/README.md]: One backtick please.
And "Defaults to ..."

Comment (LubosRemplik) [inline docs/README.md]: Fixed. Sorry about silly errors.

**Comment by dereuromark** [comment]
Ops, the tests now fail.

**Comment by LubosRemplik** [comment]
I guess I have broken unit tests by this PR, any advice? Not really familiar with PHP unit tests but will try to fix. Thanks

**Comment by dereuromark** [comment]
Well, the path is manually set there, and now always set to null (default) I assume, so you just have to fix that part.

**Comment by dereuromark** [comment]
I think I fixed it: https://github.com/dereuromark/cakephp-tinyauth/commit/8f3cb53ed1753a3e44d825ff72863745600e519f

**Comment by LubosRemplik** [comment]
Still seems to fail :/

**Comment by LubosRemplik** [comment]
Tried to find and fix the tests, but no success so far. Might take a look tomorrow.

**Comment by LubosRemplik** [comment]
Seems you did it. Thanks and sorry

**Comment by dereuromark** [comment]
Thank you!

**Pull request merged**